src/BuiltInTools/HotReloadAgent.WebAssembly.Browser/Microsoft.DotNet.HotReload.WebAssembly.Browser.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <!--
+      Intentionally pinned. Supports Visual Studio in-proc agent client.
+    -->
+    <TargetFramework>$(SdkTargetFramework)</TargetFramework>
```

Comment:
SdkTargetFramework is the latest, no? Is the comment wrong or the TFM?

Reply:
The comment is wrong